ding/utils/render_helper.py

```diff
@@ -0,0 +1,40 @@
+def render(env, render_mode=('rgb_array')):
+    '''
+    Overview:
+        Render the environment's current frame.
+    Arguments:
+        - env (:obj:`BaseEnv`): the ding env.
+        - render_mode (:obj:`str`): render mode.
+    Returns:
+        - frame (:obj:`numpy.ndarray`): [H * W * C]
+    '''
+    # TODO: do not use private member _env
+    if hasattr(env._env, 'sim'):
+        # mujoco
+        return env._env.sim.render(camera_name='track', height=128, width=128)[::-1]
+    else:
+        # other
+        return env._env.render(mode=render_mode)
+
+
+def fps(env_manager):
+    '''
+    Overview:
+        Render the environment's fps.
+    Arguments:
+        - env (:obj:`BaseEnvManager`): the ding env manager.
+    Returns:
+        - fps (:obj:`int`).
+    '''
+    # TODO: do not use private member _env_ref
+    env_ref = env_manager._env_ref
+    if hasattr(env_ref, 'model'):
+        # mujoco
```

Comment:
mujoco model-based model env?

Reply:
It refers to mujoco_py.cymj.PyMjModel.

In [1]: import gym

In [2]: env = gym.make('HalfCheetah-v2')

In [3]: env.model
Out[3]: <mujoco_py.cymj.PyMjModel at 0x55d59174a100>

Reply:
bug fixed

Reply:
It was actually a bug. It should be a reference to openai gym env instance (the return from `gym.make(*)`) and fixed in the new commit. See [render_helper.py](https://github.com/ZHZisZZ/DI-engine/blob/render/ding/utils/render_helper.py)